src/plugins/analysis/input_vectors/Dockerfile

```diff
@@ -0,0 +1,113 @@
+# installs the plugin input_vectors with radare2 in Docker
+# based on the official radare docker image
+# @ https://github.com/radare/radare2/blob/master/Dockerfile
+
+# r2docker
+# ========
+#
+# Requires 1GB of free disk space
+#
+# Build docker image with:
+# $ docker build -t r2docker:latest .
+#
+# Run the docker image:
+# $ docker images
+# $ export DOCKER_IMAGE_ID=$(docker images --format '{{.ID}}' -f 'label=r2docker')
+# $ docker run -ti --cap-drop=ALL r2docker:latest
+#
+# Once you quit the bash session get the container id with:
+# $ docker ps -a | grep bash
+#
+# To get into that shell again just type:
+# $ docker start -ai <containedid>
+#
+# To share those images:
+# $ docker export <containerid> | xz > container.xz
+# $ xz -d < container.xz | docker import -
+#
+#
+# If you willing to debug a program within Docker, you should run it with CAP_SYS_PTRACE:
+#
+# $ docker run -it --cap-drop=ALL --cap-add=SYS_PTRACE r2docker:latest
+# $ r2 -d /bin/true
+#
+
+# Using debian 9 as base image.
+FROM debian:9
+
+MAINTAINER Thomas Barabosch <thomas.barabosch@fkie.fraunhofer.de>
+
+# Label base
+LABEL r2docker latest
+
+# Radare version
+ENV R2_VERSION master
+# R2pipe python version
+ENV R2_PIPE_PY_VERSION 0.8.9
+# R2pipe node version
+ENV R2_PIPE_NPM_VERSION 2.3.2
+
+# Build radare2 in a volume to minimize space used by build
+VOLUME ["/mnt"]
+
+# Install all build dependencies
+# Install bindings
+# Build and install radare2 on master branch
+# Remove all build dependencies
+# Cleanup
+RUN DEBIAN_FRONTEND=noninteractive dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y \
+  curl \
+  gcc \
+  git \
+   python3 \
+  python-pip \
+  python3-pip \
+  bison \
+  pkg-config \
+  make \
+  glib-2.0 \
+  libc6:i386 \
+  libncurses5:i386 \
+  libstdc++6:i386 \
+  gnupg2 \
+  sudo && \
+  curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
+  apt-get install -y nodejs python-pip && \
+  pip install r2pipe=="$R2_PIPE_PY_VERSION" && \
```

Comment:
Why install r2pipe for python 2 and 3? Are both getting used?

Reply:
This container is based on one of the official radare2 ones. I am hesitated to change stuff since this may lead to problems.